code/api/src/modules/crate/model.js

```diff
@@ -7,12 +7,16 @@ module.exports = function(sequelize, DataTypes) {
     },
     description: {
       type: DataTypes.TEXT
+    },
+    deliveryDate: {
```

Comment:
Thank you for adding this property into our crate model, we will now be able to access the delivery date on the front end!